p4_symbolic/util/io.h

```diff
@@ -29,7 +29,7 @@
 namespace p4_symbolic {
 namespace util {
 
-// Read the entire content of the file and store it in "output".
+// Read the entire content of the file and return it (or an error status).
```

Comment:
One more nit regarding comments from the C++ style guide:

> [Function] comments should open with descriptive verbs in the indicative mood ("Opens the file") rather than verbs in the imperative ("Open the file"). The comment describes the function; it does not tell the function what to do. In general, these comments do not describe how the function performs its task. Instead, that should be left to comments in the function definition.

So use "Reads" instead of "Read" here.